sass/_FieldTable.scss

```diff
@@ -65,13 +66,15 @@
 
 .coveo-field-table-toggle-icon-up {
   position: absolute;
-  bottom: 10px;
+  bottom: 35px;
   right: 10px;
   margin: 0;
+  @include transition(opacity 0s);
   @include retina-sprite($arrow-up-group);
-  display: none;
+  opacity: 0;
```

Comment:
I fixed that by adding a height attribute to the component. The height is set to 0px when it is hidden, and then back to 13px when it is visible.